### DISCUSSION TITLE
How to access the admin panel when running client locally?

### DISCUSSION THREAD

**Discussion by blairmacintyre** [mozilla/hubs#4716] [Hubs Cloud Questions] [answered]
I'm trying to run the client locally against my hubs instance, and want to access the admin panel to do some debugging of it.

On discord, @netpro2k pointed me to the admin subdirectory, and said I needed to run that locally as well. 

I'm not sure how to configure it (and the hubs client) to run it locally. I know what to set for the reticulum server in the admin/.env, but not the rest of the parameters.

Aside from admin, the rest of the client works locally.

**Comment by pattersonbl2** [comment]
We don't really support the local installs outside the doumenation and readme found in Github and Hub.mozilla.com. due to our team size at the moment.  I have found that people in the community have set this up before and maybe be able to help.

**Comment by blairmacintyre** [comment, marked as answer]
I wasn't asking for "support" as much as any hints at all.  The problem here should be trivial to fix (clearly, the input and output in this field are not being handled correctly), and I admit it's incredibly frustrating to not be able to do this when it's completely borked my server.

**Reply by blairmacintyre** [reply]
@Pattersonbl2 FYI, I don't need this anymore.  @netpro2k gave me a simple (partial-but-good-enough-to-get-me-up-and-running-again) fix for the admin panel issue, so I'm good now.  It would be good if this was possible, tho!
